node_js/index.js

```diff
@@ -24,17 +28,37 @@ const ENV_BRANCH = 'development';
 
 const getArgs = () => {
   const args = parseArgs(process.argv);
+  const repository = args.repository || args.r || '';
   return {
-    repository: args.repository || args.r || '',
+    repository,
     tech: args.tech || args.t || NODE_TECH,
     projectName: args.projectName || args.p || '',
     branch: args.branch || args.b || ENV_BRANCH,
-    metricsUrl: args.metricsUrl || args.m || NODE_METRICS_URL
+    metricsUrl: args.metricsUrl || args.m || NODE_METRICS_URL,
+    elasticApmProject: args['elastic-apm-project'] || repository
```

Comment:
how you pass this args when you run the script?, I recommended passing a shorthand similar to`eap`

Reply:
`--elastic-apm-project=<project name>`

Reply:
Shorthand must contain only one letter :(